src/pyqasm/entrypoint.py

```diff
@@ -30,11 +30,15 @@
     import openqasm3.ast
 
 
-def load(filename: str) -> QasmModule:
+def load(
+    filename: str, device_qubits: int | None = None, consolidate_qubits: bool = False
```

Comment:
Please add these add `kwargs` and keep `consolidate_qubits` on the `unroll` level